### DISCUSSION TITLE
Meta: Include a section in readme about not opening new issues

### DISCUSSION THREAD

**Discussion by peteris-zealid** [MatthiasValvekens/pyHanko#74] [General]
Hello Matthias, I noticed that you often have the problem that people open new issues when you would actually like them to create a new "discussion". I was thinking that adding a section to the readme file could mitigate the problem. Something like 'contributing' or 'how to get help'. The latter might not be that good because it implies that users are entitled to get help. That being said I am appriciate your willingness to help people.

**Comment by MatthiasValvekens** [comment]
Good point; I put it in the `CONTRIBUTING` file, but you're right that an additional call-out in the readme wouldn't hurt. I'll add a couple words there, thanks!

**Reply by MatthiasValvekens** [reply]
For whatever reason it's not showing up here, but I did just that in 3a538ea.
